packages/matchbox/src/components/ThemeProvider/theme.js

```diff
@@ -164,8 +164,9 @@ const theme = {
 
   // Example: <Box border={500} />
   borders: {
+    300: `${tokens.borderWidth_100} solid ${tokens.color_gray_300}`,
     400: `${tokens.borderWidth_100} solid ${tokens.color_gray_400}`,
-    500: `${tokens.borderWidth_200} solid ${tokens.color_gray_400}`,
+    500: `${tokens.borderWidth_200} solid ${tokens.color_gray_300}`,
```

Comment:
Also need to make the same changes in `components/SystemPropsTable/data.js` so the site docs match up